src/core_modules/capture-core/components/ScopeSelector/QuickSelector/OrgUnitSelector.component.js

```diff
@@ -1,185 +1,77 @@
-                        <h4 className={this.props.classes.title}>{ i18n.t('Selected registering unit') }</h4>
-                        <div className={this.props.classes.selectedItemContainer}>
-                            <div>{this.props.selectedOrgUnit.name}</div>
-                            <div className={this.props.classes.selectedItemClear}>
-                                <IconButton data-test="reset-selection-button" className={this.props.classes.selectedButton} onClick={() => this.props.onReset()}>
-                                    <ClearIcon className={this.props.classes.selectedButtonIcon} />
-                                </IconButton>
-                            </div>
-                        </div>
-                    </Paper>
-                    <Dialog
-                        open={this.state.open}
-                        onClose={this.handleClose}
-                    >
-                        <DialogTitle>{i18n.t('Start Again')}</DialogTitle>
-                        <DialogContent>
-                            <DialogContentText>
-                                {i18n.t('Are you sure? All unsaved data will be lost.')}
-                            </DialogContentText>
-                        </DialogContent>
-                        <DialogActions>
-                            <Button
-                                onClick={this.handleClose}
-                                secondary
-                            >
-                                {i18n.t('Cancel')}
-                            </Button>
-                            <Button
-                                onClick={this.handleReset}
-                                primary
-                            >
-                                {i18n.t('Accept')}
-                            </Button>
-                        </DialogActions>
-                    </Dialog>
-                </div>
-            );
-        }
+        const { selectedOrgUnitId, selectedOrgUnit, previousOrgUnitId, onReset, classes } = this.props;
 
         return (
-            <div data-test="org-unit-selector-container">
-                <Paper square elevation={0} className={this.props.classes.paper}>
-                    <h4 className={this.props.classes.title}>{ i18n.t('Registering Organisation Unit') }</h4>
-                    <div>
-                        <OrgUnitField
-                            data-test="org-unit-field"
-                            onSelectClick={this.handleClick}
-                            previousOrgUnitId={this.props.previousOrgUnitId}
-                        />
-                    </div>
-                </Paper>
-            </div>
+            <SelectorBarItem
+                displayOnly={Boolean(selectedOrgUnitId)}
+                label={i18n.t('Registering unit')}
+                noValueMessage={i18n.t('Choose a registering unit')}
+                value={Boolean(selectedOrgUnitId) && selectedOrgUnit?.name}
```

Comment:
The component throws an error here when there is no value - we're passing in a bool, while it expected a string. Does this have something to do with the closing issue mentioned above?

![image](https://user-images.githubusercontent.com/12842868/212643424-094057db-84ed-4420-a64d-e0ee4423ba71.png)